api/nakadi-event-bus-api.yaml

```diff
@@ -34,22 +99,252 @@ paths:
       summary: Get monitoring metrics
       responses:
         '200':
-          description: Metrics data
+          description: Ok
           schema:
             $ref: '#/definitions/Metrics'
         '401':
-          description: User is not authenticated
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
           schema:
             $ref: '#/definitions/Problem'
         '503':
           description: Not available
           schema:
             $ref: '#/definitions/Problem'
+  /event-types:
+    get:
+      tags:
+        - schema-registry-api
+      description: Returns a list of all registered `EventType`s
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/EventType'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - schema-registry-api
+      description: |
+        Creates a new `EventType`. Upon creation the name MUST be unique. Failing to do so will fail the
+        request with a 422 Unprocessable entity.
+
+        **Implementation note:** The creation of an EventType implicitly creates the structures in the backing
+        messaging implementation needed for the reception and persistency of the Events. Considering that at
+        this time only Kafka is used, this corresponds to the creation of a Topic. If so desired, clients can
+        interact directly with the topic using the low level API (for existing restrictions see the
+        corresponding methods on the topic-api).
```

Comment:
I think I would maybe separated these two steps, because of SLAs and keeping control over the guaranteed scalability for one Topic (EventType, respectively).